README.md

```diff
@@ -1,2 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # capstone-1
 Building an online website for a conference
+
+![screenshot](./images/2021-10-12.png)
+
+.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Application Page url
+
+https://phelian23.github.io/capstone-1/
```

Comment:
- [ ] The link is not working, please update it.

![jjjjj](https://user-images.githubusercontent.com/61562626/137153236-0ec3ea84-28b4-4e94-8c39-32106e640ddd.jpeg)